MSAL/IdentityCore

```diff
@@ -1 +1 @@
-Subproject commit 2496b06403af6bdecaccbaefc6dcfea34c16c1f6
```

Comment:
This pull request does not update CHANGELOG.md.

Please consider if this change would be noticeable to a partner or user and either update CHANGELOG.md or resolve this conversation.
<!-- changelog-comment -->